### DISCUSSION TITLE
调用 router.push 问题

### DISCUSSION THREAD

**Discussion by chuangmaster** [un-pany/v3-admin-vite#232] [General]
作者您好，谢谢您的专案，让我有学习机会
我尝试在登出按钮调用调用 router.push("/logout")
发现画面都不会进行跳转
最后发现 router.beforeEach(async (to, _from, next)
都会再次进到 /login -> /dashboard
想请教您原因为何?

**Comment by pany-ang** [comment]
检查一下是不是 token 没有清除
